pmg/static/resources/css/style.scss

```diff
@@ -213,6 +234,10 @@ article.page {
   img {
     margin-bottom: 10px;
   }
+
+  @media (max-width:768px) {
+    margin-top: 10px;
+  }
 }
 
 .correct-this-page-footer {
```

Comment:
Alternatively, let's just give .footer-1 some margin-top. The idea is to stop the content from being flush against it.